csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst/MakeConstAnalyzer.cs

```diff
@@ -28,10 +28,11 @@ public class MakeConstAnalyzer : DiagnosticAnalyzer
 
         public override void Initialize(AnalysisContext context)
         {
-            // See https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Analyzer%20Actions%20Semantics.md for more information
-            // <SnippetRegisterNodeAction>
```

Comment:
The open snippet tag cannot be deleted or moved (as it is in line 35 in this case). This causes the snippet to disappear from the build and generates a build warning.

Reply:
This is the same issue I fixed in VB the comment was moved by a CodeFix. I committed a fix, but the snippet should be 3 lines not 1
```cs
        public override void Initialize(AnalysisContext context)
        {
             // See https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Analyzer%20Actions%20Semantics.md for more information
             // <SnippetRegisterNodeAction>
            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.None);
            context.EnableConcurrentExecution();
            context.RegisterSyntaxNodeAction(AnalyzeNode, SyntaxKind.LocalDeclarationStatement);
            // </SnippetRegisterNodeAction>
        }
```